macros/models/_sector/nft/nft_mints.sql

```diff
@@ -0,0 +1,181 @@
+        , nft_mints.tx_hash AS tx_hash
+        , etxs."from" AS tx_from
+        , etxs.to AS tx_to
+        , UINT256 '0' AS platform_fee_amount_raw
+        , DOUBLE '0' AS platform_fee_amount
+        , DOUBLE '0' AS platform_fee_amount_usd
+        , DOUBLE '0' AS platform_fee_percentage
+        , CAST(NULL as VARBINARY) AS royalty_fee_receive_address
+        , '0' AS royalty_fee_currency_symbol
+        , UINT256 '0' AS royalty_fee_amount_raw
+        , DOUBLE '0' AS royalty_fee_amount
+        , DOUBLE '0' AS royalty_fee_amount_usd
+        , DOUBLE '0' AS royalty_fee_percentage
+        , nft_mints.evt_index
+        FROM {{ ref('nft_transfers') }} nft_mints
+        LEFT JOIN nfts_per_tx nft_count ON nft_count.tx_hash=nft_mints.tx_hash
+        LEFT JOIN {{ base_traces }} et ON et.block_time=nft_mints.block_time
+            AND et.tx_hash=nft_mints.tx_hash
+            AND et."from"=nft_mints.to
+            AND (et.call_type NOT IN ('delegatecall', 'callcode', 'staticcall') OR et.call_type IS NULL)
+            AND et.success
+            AND CAST(et.value as DOUBLE) > 0
+            {% if is_incremental() %}
+            AND {{incremental_predicate('et.block_time')}}
+            {% endif %}
+        LEFT JOIN {{ source('prices','usd') }} pu_eth 
+            ON pu_eth.blockchain IS NULL
+            AND pu_eth.minute=date_trunc('minute', et.block_time)
+            AND pu_eth.symbol = 'ETH'
+            {% if is_incremental() %}
+            AND {{incremental_predicate('pu_eth.minute')}}
+            {% endif %}
+        LEFT JOIN {{ erc20_transfer }} erc20s ON erc20s.evt_block_time=nft_mints.block_time
+            AND erc20s."from"=nft_mints.to
+            AND erc20s.evt_tx_hash = nft_mints.tx_hash
+            AND (et.value IS NULL OR CAST(et.value as double) = 0)
+            {% if is_incremental() %}
+            AND {{incremental_predicate('erc20s.evt_block_time')}}
+            {% endif %}
+        LEFT JOIN {{ source('prices','usd') }} pu_erc20s ON pu_erc20s.blockchain='{{blockchain}}'
+            AND pu_erc20s.minute=date_trunc('minute', erc20s.evt_block_time)
+            AND erc20s.contract_address=pu_erc20s.contract_address
+            {% if is_incremental() %}
+            AND {{incremental_predicate('pu_erc20s.minute')}}
+            {% endif %}
+        LEFT JOIN {{ base_transactions }} etxs ON etxs.block_time=nft_mints.block_time
+            AND etxs.hash=nft_mints.tx_hash
+            {% if is_incremental() %}
+            AND {{incremental_predicate('etxs.block_time')}}
+            {% endif %}
+        LEFT JOIN {{ ref('nft_aggregators') }} agg 
+            ON etxs.to=agg.contract_address
+            AND agg.blockchain = '{{blockchain}}'
+        LEFT JOIN {{ ref('tokens_nft') }} tok 
+            ON tok.contract_address=nft_mints.contract_address
+            and tok.blockchain = '{{blockchain}}'
+        LEFT JOIN namespaces ec ON etxs.to=ec.address
+        {% if is_incremental() %}
```

Comment:
@chuxinh I think this block is the cause of the incremental error.
I have not seen a usage of `{{this}}` in a macro before, and it might resolve to the the macro name instead of the model name. :thinking: 
I'll try running this without this block.

Reply:
> I have not seen a usage of {{this}} in a macro before, and it might resolve to the the macro name instead of the model name. 🤔
Ah this is like joining to the existing table, I think i've used it a few times in spellbook